components/hero/styled-components.ts

```diff
@@ -4,10 +4,9 @@ import Typography from '@mui/material/Typography';
 
 import Image from 'next/image';
 
-export const PageContainer = styled('div')`
+export const PageContainerSecondary = styled('div')`
```

Comment:
It would be great if we don't need to create another component

Reply:
'+